src/backend/api/config.py

```diff
@@ -20,7 +20,10 @@
 
 
 class ConfigResponse(BaseModel):
-    endpoint: str
+    Endpoint: str
+    InstanceId: str
+    Delimiter: str
+    ApiPath: str
```

Comment:
I am using CamelCase here because we are using that in the S3 API everywhere. To do not have a mixture of various styles we should stay with one in the whole app IMO.